bugbug/tools/code_review.py

```diff
@@ -680,6 +717,34 @@ def collect_function_definitions(function_declarations, target_element, definiti
         )
 
 
+def gather_line_context(line_context):
+    """Reformat the line context list and remove duplicates.
+
+    Args:
+        line_context: List of lists, where each list is [line, file, function].
```

Comment:
Is the received input (i.e., `line_context`) a list of lists or a list of tuples?